packages/tenant-process/src/services/tenantService.ts

```diff
@@ -353,6 +357,77 @@ export function tenantServiceBuilder(
       return updatedTenant;
     },
 
+    async addDeclaredAttribute({
+      tenantAttributeSeed,
+      authData,
+      correlationId,
+    }: {
+      tenantAttributeSeed: ApiDeclaredTenantAttributeSeed;
+      authData: AuthData;
+      correlationId: string;
+    }): Promise<Tenant> {
+      logger.info(
+        `Add declared attribute ${tenantAttributeSeed.id} to requester tenant ${authData.organizationId}`
+      );
+      const targetTenant = await retrieveTenant(
+        authData.organizationId,
+        readModelService
+      );
+
+      const maybeDeclaredTenantAttribute = targetTenant.data.attributes.find(
+        (attr) => attr.id === tenantAttributeSeed.id
+      );
```

Comment:
Probably we could add the filter for the type, like [here](https://github.com/pagopa/interop-be-monorepo/pull/407/files#diff-c52f69aae83f1d150aa67f380d299976f1a67b9cb76738048267062cdb848f3eR588-R592)